meetings/2024-10-02.md

```diff
@@ -0,0 +1,109 @@
+* eP: Instead of making up use cases, we can work with existing communities, preferably open source and document their real world usecases which already have user base. As an example we had a meeting with https://openfoodnetwork.org/
+* eP: We can also try to "eat our own cookings" more. This prototype tries to define some CG meetings based requirements https://github.com/janeirodigital/sai-js/tree/main/examples/plenary#requirements
+* eP: Unfortunately I didn't participate in the TPAC UC meeting.
+* eP: We (CG) has a bunch of work on this already (linked above).
+* eP: We need to have UCR managed in a way to evaluate proposals. Do we do it by simplicity or emotions or something else? ??? objectively. We can have a matrix for the proposals, then we can see best on practice. With e.g., added complexity, we could do with simpler. This is because different people coming with different proposals. Once we have implementation experience it is going to be subjective. We can have some rough way of evaluating the complexity. For me the main point for the UCs is to evaluate all the proposals. We are kind of stuck on what to focus on.
+* RG: I like a lot of the ideas behind on having some process to make a note of it, but at the end of the day we could do the matrices etc but it'll be subjective and no real way of measuring complexity. It is more of art than science.
+* MdJ: We can measure complexity by looking at how many implementing. Adding traction as a criteria.
+* RG: That's one measure and so many different ways of measuring. It is not something we can solve for, but to have an intuition for.
+* MdJ: If you look to see what solutions get traction, that is a proxy for how complex they are to implement. Look at shape files for instance, very few app devs write them, I think that's an indication that it's quite a complex solution.
+* SC: Use cases are important. In the past we had a use case survery, which ties to Michiel's point. This can signal from the community interest and commitment. People are brining up age old issues about Solid should be doing this or that. We have an aboundance of use cases. Data portability comes up, access requests. We had issues, we had proposals. For example [data migrator](https://pdsinterop.org/solid-migrator/). I'm also part of this and we didn't do great job on evaluating those proposals. We need storage provisiooning, migrating storate. There are proposals and we don't give them enough attention. Later in the future we come back "oh we need that". We do need those use cases but there are solutions in front of us. A solution is better than no solution. Recently we heard need for digital wallets, sometimes it is hard to dig out those proposals. We need to be evaluating those solutions, if we don't them enough attention people get impression that we don't take them seriously. I don't want us to get lost in the process and loos track of simple solutions in front of us.
```

Comment:
```suggestion
* SC: Use cases are important. In the past we had a use case survey, which ties to Michiel's point. This can signal interest and commitment from the community. People are bringing up age old issues about "Solid should be doing this or that". We have an abundance of use cases. Data portability comes up, access requests. We had issues, we had proposals. For example, [data migrator](https://pdsinterop.org/solid-migrator/). I'm also part of this, and we didn't do great job on evaluating those proposals. We need storage provisioning, storage migration. There are proposals, and we don't give them enough attention. Later in the future we come back, "oh, we need that". We do need those use cases, but there are solutions in front of us. A solution is better than no solution. Recently we heard a need for digital wallets; sometimes it is hard to dig out those proposals. We need to be evaluating those solutions. If we don't give them enough attention, people get the impression that we don't take them seriously. I don't want us to get lost in the process and lose track of simple solutions in front of us.
```